app/forms/idv/doc_pii_form.rb

```diff
@@ -25,68 +38,54 @@ def submit
         success: valid?,
         errors: errors,
         extra: {
-          pii_like_keypaths: [[:pii]], # see errors.add(:pii)
+          pii_like_keypaths: self.class.pii_like_keypaths,
           attention_with_barcode: attention_with_barcode?,
         },
       )
       response.pii_from_doc = pii_from_doc
       response
     end
 
-    private
-
-    attr_reader :pii_from_doc
-
-    def validate_pii
-      if error_count > 1
-        errors.add(:pii, generic_error, type: :generic_error)
-      elsif !name_valid?
-        errors.add(:pii, name_error, type: :name_error)
-      elsif dob.blank?
-        errors.add(:pii, dob_error, type: :dob_error)
-      elsif !dob_meets_min_age?
-        errors.add(:pii, dob_min_age_error, type: :dob_min_age_error)
-      elsif address1.blank?
-        errors.add(:pii, address_error, type: :address_error)
-      elsif !state_valid?
-        errors.add(:pii, generic_error, type: :generic_error)
-      elsif !zipcode_valid?
-        errors.add(:pii, generic_error, type: :generic_error)
-      elsif !jurisdiction_valid?
-        errors.add(:pii, generic_error, type: :generic_error)
+    def self.pii_like_keypaths
+      keypaths = [[:pii]]
+      attrs = %i[name dob dob_min_age address1 state zipcode jurisdiction]
+      keypaths << attrs
+      attrs.each do |k|
+        keypaths << [:errors, k]
+        keypaths << [:error_details, k]
       end
+      keypaths
     end
```

Comment:
Just for my own note,  the pii_like_keypaths looks like following now, this explicitly tells `FakeAnalytics` that this nested hash keys are allowed entry when invoking `track_event` and there is no pii info exposed.

`[[:pii],
 [:name, :dob, :dob_min_age, :address1, :state, :zipcode, :jurisdiction],
 [:errors, :name],
 [:error_details, :name],
 [:errors, :dob],
 [:error_details, :dob],
 [:errors, :dob_min_age],
 [:error_details, :dob_min_age],
 [:errors, :address1],
 [:error_details, :address1],
 [:errors, :state],
 [:error_details, :state],
 [:errors, :zipcode],
 [:error_details, :zipcode],
 [:errors, :jurisdiction],
 [:error_details, :jurisdiction]]`